acct-user/gvm/gvm-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Greenbone vulnerability management program user"
+ACCT_USER_ID=495
+ACCT_USER_HOME=/var/lib/gvm
+ACCT_USER_SHELL=/bin/bash
+ACCT_USER_GROUPS=( gvm )
```

Comment:
You also need acct-group/ for this.

Reply:
Added acct-group/gvm with seperate commit.